dsl/properties/scripts/BaseClient.groovy

```diff
@@ -1,5 +1,7 @@
 @Grab(group='net.sf.json-lib', module='json-lib', version='2.3', classifier='jdk15')
 @Grab('org.codehaus.groovy.modules.http-builder:http-builder:0.7.1')
+@Grab(group='xml-apis', module='xml-apis', version='1.4.01')
+@Grab('org.yaml:snakeyaml:1.19')
```

Comment:
The grape dependency jars are bundled within the plugin since the agent machines where the procedure steps run may not always have access to external repositories for downloading the grape dependencies at runtime. So, we will need to bundle the yaml jars. See for reference: https://github.com/electric-cloud/EC-Kubernetes/tree/master/lib/grapes/org.yaml/snakeyaml